gnovm/pkg/gnolang/values.go

```diff
@@ -2132,6 +2132,11 @@ func (tv *TypedValue) GetLength() int {
 			return bt.Len
 		case *SliceType:
 			return 0
+		case *PointerType:
+			if at, ok := bt.Elt.(*ArrayType); ok {
+				return at.Len
+			}
+			panic(fmt.Sprintf("unexpected pointer type for len(): %s", tv.T.String()))
```

Comment:
Can you follow code coverage annotations and add test cases for failing scenarios?

Reply:
Added the test to check for other panic messages as well.

https://github.com/gnolang/gno/pull/2709/commits/af7364186e2aea805b41350320a477f371da3a57